glide-core/src/client/value_conversion.rs

```diff
@@ -411,11 +442,24 @@ fn convert_array_elements(
     Ok(Value::Array(converted_array))
 }
 
-fn convert_array_to_map(
```

Comment:
I'm a bit confused about the expected input format, to me it looks like this function converts 2d or 1d array representations of maps, eg [[key1, value1], [key2, value2]] or [key1, value1, key2, value2]? If it converts either we might want to clarify that

Reply:
I've added some documentation.  Hopefully that makes it better.